packages/ra-ui-materialui/src/detail/TabbedShowLayoutTabs.tsx

```diff
@@ -42,9 +42,14 @@ export const TabbedShowLayoutTabs = ({
     );
 };
 
+// @rattrayalex this is a case of nested ternaries being forced to break.
 export const getTabFullPath = (tab, index, baseUrl) =>
     `${baseUrl}${
-        tab.props.path ? `/${tab.props.path}` : index > 0 ? `/${index}` : ''
+        tab.props.path ?
+            `/${tab.props.path}` :
+        index > 0 ?
+            `/${index}` :
+            ''
```

Comment:
This demonstrates the fact that we now force all nested ternaries to break, even if they'd fit on one line. Thoughts?